admin_manual/installation/linux_installation.rst

```diff
@@ -3,27 +3,70 @@ Preferred Installation Method
 =============================
 
 For production environments, we recommend the installation from the tar archive. 
-This applies in particular to scenarios, where the web server, storage and database are on separate machines. 
+This applies in particular to scenarios, where the Web server, storage and database are on separate machines. 
+In this constellation, all dependencies and requirements are managed by the package management 
+of your operating system, while the ownCloud code itself is maintained in a sequence of simple steps 
+as documented in our instructions for the :doc:`Manual Installation on Linux <source_installation>` and the :doc:`Manual ownCloud Upgrade <../maintenance/manual_upgrade>`.
 
-In this constellation, all dependencies and requirements are managed by your operating system's package manager, while the ownCloud code itself is maintained in a sequence of simple steps as documented in our instructions for the :doc:`Manual Installation on Linux <source_installation>` and the :doc:`Manual ownCloud Upgrade <../maintenance/manual_upgrade>`.
 The package installation is for single-server setups only.
 
-Repositories
-------------
+Changes in the Linux Distribution Packages 9
+--------------------------------------------
+
+Linux distribution packages (from `Open Build Service`_) have been divided into 
+multiple packages for ownCloud 9: ``owncloud``, ``owncloud-deps`` and ``owncloud-files``. 
+
+- **owncloud:** This package installs ownCloud, complete with all dependencies.
+- **owncloud-files:** This package installs only ownCloud. It does not install *Apache*, *a database*, or any *PHP dependencies*. 
+- **owncloud-deps:** This packages install only ownCloud's dependencies (*Apache*, *PHP*, and *MySQL*). It is not intended to be installed by itself, but rather is pulled in by the ``owncloud`` metapackage. 
+
+``owncloud-files`` is available for the following distributions, but not ``owncloud-deps``.
+
+You will have to install your own LAMP stack first. This 
+allows you to create your own custom LAMP stack without dependency conflicts 
+with the ownCloud package. Browse 
+`<http://download.owncloud.org/download/repositories/9.1/owncloud/>`_ to find 
+the ``owncloud-files`` package for your distro:
+
+* Ubuntu 14.04, 16.04
+* Debian 7, 8
+* RHEL 6, 7
+* CentOS 6 SCL, 7
+* SLES 12, 12 SP1
+* openSUSE 13.2, Leap 42.1
+
+ownCloud packages with dependencies are available for the following Linux distributions:
 
-You may use either of the following repositories for ownCloud 9.1:
+* Ubuntu 14.04, 16.04
+* Debian 8
+* RHEL 7
+* CentOS 7
+* SLES 12
+* openSUSE 13.2, Leap 42.1
+
```

Comment:
This distro list is outdated. Need to find time to update this at some point. Maybe you could help? Actual policy is that we support the latest two versions and previous LTS (in ubuntu terms this would be 17.04, 16.10 and 16.04 LTS).